base/methodshow.jl

```diff
@@ -240,17 +240,17 @@ function show_method_table(io::IO, ms::MethodList, max::Int=-1, header::Bool=tru
     end
     n = rest = 0
     local last
-    LAST_SHOWN_LINE_INFOS = get(io, :LAST_SHOWN_LINE_INFOS, Tuple{String,Int}[])
+    last_shown_line_infos = get(io, :last_shown_line_infos, Tuple{String,Int}[])
 
-    resize!(LAST_SHOWN_LINE_INFOS, 0)
+    resize!(last_shown_line_infos, 0)
```

Comment:
What's the point in modifying `last_shown_line_infos` when it isn't provided by the `io`? The result will never be used in that case, right?

Reply:
Yes, you are correct, this was just the easiest way to implement this at the time. Do you think it's worth refactoring this logic?

Reply:
I think it would be better to do 

```
last_linfo = get(io, :last_shown_line_infos, nothing)
...
last_linfo === nothing || push!(last_shown_line_infos, (string(file), line))
```

Is the `resize!` needed now? There is always a length zero coming from the `io`, no?

Reply:
I think we still need `resize!` since you can print multiple stack traces in one `io`. Previous discussion:

https://github.com/JuliaLang/julia/pull/35556#discussion_r435535024
https://github.com/JuliaLang/julia/pull/35556#issuecomment-643663448